vuepress/docs/next/tutorials/samples/install-standard-demo.md

```diff
@@ -8,18 +8,29 @@ sidebarDepth: 2
 
 ## Introduction
 
-This tutorial will guide you through installing a demo application using the Entando Component Repository and a set of Entando
-bundles. This solution template includes microservices, micro frontends, multiple pages, and CMS content. The goal of this exercise
-is to demonstrate how Entando bundles can be used to quickly install and create functionality in an Entando Application, enable portable business capabilities, and allow developers to reuse full stack operations via bundles.
+This tutorial will guide you through installing a demo application using the Entando Component Repository (ECR) and a set of Entando
+bundles. This solution template includes 
```

Comment:
does it need a colon or comma

Reply:
definitely not a comma. i thought we opted to nix the colon and jump right into bullets? i can revert to colons but prefer not to have one; it almost acts as an interruption.

Reply:
I'm don't have a strong opinion on this one. I think I like the forewarning that a list is coming.